### DISCUSSION TITLE
AMA with CS50's David Malan and GitHub Education Director of Product Divya Vaishnavi

### DISCUSSION THREAD

**Discussion by elisemoe** [community/community#18276] [GitHub Education]
![image](https://user-images.githubusercontent.com/6633808/173104332-20348145-bdc1-415d-9631-ec843f6e939a.png)

Join us now on Twitch to hear from GitHub Education Professor-in-Residence (and CS50 instructor) to talk about rubber duck debugging, using GitHub Classroom and Codespaces, and more 📚 📺

https://www.twitch.tv/githubeducation

Miss the stream? Post your questions for Divya and David here! ⬇️

**Comment by xiu-shi** [comment]
Would love to hear some success stories of our fellow students who managed transformed career in tech and their learning experience.

**Reply by dmalan** [reply]
Here's one you might find of interest!

https://medium.com/geekculture/how-i-went-from-professional-chef-to-a-self-taught-full-stack-developer-in-one-year-my-story-168b4e339189

**Reply by xiu-shi** [reply]
Thanks @dmalan

**Comment by mohassan-dev** [comment]
Hi David and Divya!

What do you think students should do after finishing CS50X and CS50web and CS50P?

**Reply by dmalan** [reply]
If AI is of interest, you might enjoy https://cs50.edx.org/ai, or if game development, https://cs50.edx.org/games. For more on CS theory, though, you might like https://www.coursera.org/learn/algorithms-part1! 

If software engineering's your goal, though, I'd try to tackle some personal or professional project of interest as soon as an idea or opportunity arises. Odds are you'll learn even more when it's you who's directing the "assignment"!

**Reply by Blauelf** [reply]
Guttenbug (error by copy&paste), that's https://cs50.edx.org/ai twice, second link should have been https://cs50.edx.org/games ;)

**Reply by dmalan** [reply]
Oops, thanks, fixed!

**Reply by mohassan-dev** [reply]
> If AI is of interest, you might enjoy https://cs50.edx.org/ai, or if game development, https://cs50.edx.org/games. For more on CS theory, though, you might like https://www.coursera.org/learn/algorithms-part1!
> 
> If software engineering's your goal, though, I'd try to tackle some personal or professional project of interest as soon as an idea or opportunity arises. Odds are you'll learn even more when it's you who's directing the "assignment"!

Thank you David!
I will definitely check out algorithms part1 after i finish CS50 web <3

**Reply by statick88** [reply]
Oh, thanks

**Comment by VR-Francisco** [comment]
Hello! 

 If I finish a CS50x course then will I have the enough knowledge to start with an advanced course like CS50 mobile app?

**Reply by s-hackman** [reply]
Prerequisites:
 CS50 or strong background with HTML and CSS and comfort with basic JavaScript.

**Reply by Blauelf** [reply]
Note that CS50's Mobile App Development with React Native is outdated and has been discontinued mid 2020.

As [the course website](https://cs50.harvard.edu/mobile/2018/) states, the assignments will no longer be graded, you won't be able to get a certificate.

Best to find a more recent resource if you want to learn mobile app development, web frontend and mobile courses usually get updated at least every two years. Web backend is a bit more stable. Of course that other resource might have different requirements.

**Reply by VR-Francisco** [reply]
> 

That's amazing, thank you for your answer. I don't care if I don't get a certification my goal this year is to learn React Native, do you know a good place to learn? Because when I saw the CS50 for mobile I thought that it was the same that I will learn or what is it about this course? I want to learn to develop apps for android and ios phones.

**Reply by dmalan** [reply]
CS50's own mobile class is indeed outdated, and I'm not familiar with similar offerings personally, but a quick search did turn up https://medium.com/javarevisited/top-5-react-native-courses-for-mobile-application-developers-b82febdf8a46, which might help!

**Comment by Blauelf** [comment]
Is there recent data regarding how many students joined the various courses vs. how many completed them? Maybe with some more detail how far they likely got? Kind of like https://cs50.medium.com/this-was-cs50x-82be0995862b but maybe not in such detail, and for more than just CS50x.

**Reply by dmalan** [reply]
Not that we've published ourselves, but if you visit each of the landing pages for the courses at https://edx.org/cs50, you can see how many folks have joined, at least!

**Comment by Budisti** [comment]
Hello! My goal is to work in a big tech company in America but I’m not an American, can you give me a plan to follow to reach that goal.

**Reply by juanpflores** [reply]
Hi @Budisti 👋🏼 
There are several paths to getting a job in a US company. My recommendations are

1. Learn English and practice, practice, practice! I can't tell you how many folks I know lost great opportunities because of their English proficiency. If there's any way that you can join voice chats with folks from the US that's even better
2. Chose what you want to specialize in and build projects around that technology Sharing them on your GitHub can help you showcase a portfolio of products you've built :octocat: 
3. Get under people's radar. Attend and speak conferences at conferences that are related to the topic you're specializing in. If you're speaking it is even better. These events will also help you to build a network in that industry. 
4. Keep trying. Don't get frustrated with the process of applying to companies.
